addons/test_mail/static/tests/tracking_value_tests.js

```diff
@@ -118,7 +118,7 @@ QUnit.test('rendering of tracked field of type float: from 0 to non-0', async fu
     await testUtils.fields.editInput(form.$('input[name=float_field]'), 1);
     await click('.o_form_button_save');
     assert.strictEqual(
-        document.querySelector('.o_TrackingValue').textContent,
+        document.querySelector('.o_MessageTrackingValue').textContent,
```

Comment:
It's strange to have all this diff, tracking was moved outside of message, but we are now adding a message prefix on it.